### ISSUE TITLE
Crashing when sequence_ids: []

### ISSUE THREAD

**Issue by davidyuyuan** [PacificBiosciences/pb-StarPhase#12]
```
[2024-05-15T21:56:41.601Z DEBUG pbstarphase::cyp2d6_caller] Searching A00297:51:HFM5LDSXX:1:2344:6325:12727 at 42127132
[2024-05-15T21:56:41.618Z DEBUG pbstarphase::data_types::cyp2d6_typer] Collapsed calls:
[2024-05-15T21:56:41.618Z DEBUG pbstarphase::cyp2d6_caller] Found 0 regions of interest.
[2024-05-15T21:56:41.618Z DEBUG pbstarphase::cyp2d6_caller] Searching A00217:60:HCVVJDSXX:2:2606:4933:6558 at 42126466
[2024-05-15T21:56:41.634Z DEBUG pbstarphase::data_types::cyp2d6_typer] Collapsed calls:
[2024-05-15T21:56:41.634Z DEBUG pbstarphase::cyp2d6_caller] Found 0 regions of interest.
[2024-05-15T21:56:41.634Z DEBUG pbstarphase::cyp2d6_caller] Searching A00217:60:HCVVJDSXX:1:1371:13982:25113 at 42125926
[2024-05-15T21:56:41.645Z DEBUG pbstarphase::data_types::cyp2d6_typer] Collapsed calls:
[2024-05-15T21:56:41.645Z DEBUG pbstarphase::cyp2d6_caller] Found 0 regions of interest.
[2024-05-15T21:56:41.646Z DEBUG pbstarphase::cyp2d6_caller] sequence_ids: []
thread 'main' panicked at /usr/local/cargo/git/checkouts/dynamic_wfa_consensus-847e68f1d0159214/3ac4584/src/priority_consensus.rs:167:45:
index out of bounds: the len is 0 but the index is 0
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
The command to call is:
```
/opt/conda/bin/pbstarphase diplotype -v --bam "${output_dir}/${file}" -d "${CPIC_JSON_DB}" -t 16 \
    -r "${ref_genome}" -c "${vcf_file}" -o "${output_dir}/${file}.json" --pharmcat-tsv "${output_dir}/pharmcat.tsv"
```
The alignment used in this test is of 30x GRCh38 from the G1K dataset: `ftp://ftp.sra.ebi.ac.uk/vol1/run/ERR323/ERR3239279/NA07000.final.cram` downloadable from EBI.

**Comment by holtjma** [comment]
Thanks for reporting this, it's caused by no useable reads in the _CYP2D6_ region. Someone else reported this to us earlier, so there should be a patch for it with the next release.

**Comment by holtjma** [comment]
This should be resolved with v0.10.1: https://github.com/PacificBiosciences/pb-StarPhase/releases/tag/v0.10.1

**Comment by davidyuyuan** [comment]
Thanks for the prompt response. I just checked that it is not available in bioconda yet. I will verify it once its conda package is available.
